release/store_artifacts.sh

```diff
@@ -89,8 +89,12 @@ if [[ "${PUSH_DOCKER}" == "true" ]]; then
     fi
     gcloud auth configure-docker -q
     docker load -i "${TAR_PATH}"
-    docker tag "istio/${IMAGE_NAME}:${VER_STRING}" "${GCR_PATH}/${IMAGE_NAME}:${VER_STRING}"
+    echo "FROM istio/${IMAGE_NAME}:${VER_STRING}
+COPY LICENSES.txt /" > Dockerfile
```

Comment:
We should zip this (checking with legal, cc'd you).

Reply:
per discussion it seems unnecessary because of docker layer compression
/lgtm
/approve